### PR TITLE
create note

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5150

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing and style checks needed for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/tvm-dashboard-insights.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ Area | Description
 **Top remediation activities** | Track the remediation activities generated from the security recommendations. You can click each item on the list to see the details in the **Remediation** page or click **Show more** to see the rest of the remediation activities, and active exceptions.
 **Top exposed machines** | See the exposed machine names and their exposure level. You can click each machine name from the list and it will take you to the machine page where you can view the alerts, risks, incidents, security recommendations, installed software, discovered vulnerabilities associated with the exposed machines. You can also do other EDR-related tasks in it, such as: manage tags, initiate automated investigations, initiate a live response session, collect an investigation package, run antivirus scan, restrict app execution, and isolate machine. You can also click **Show more** to see the rest of the exposed machines list.
 
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/tvm-dashboard-insights.md]:
```suggestion
> [!NOTE]
```

File: windows/security/threat-protection/microsoft-defender-atp/tvm-dashboard-insights.md [hunk 2]
```diff
@@ -66,6 +66,9 @@ Area | Description
...
+>Machines with no alerts seen in the last 30 days do not count towards the exposure score of Threat & Vulnerability Management.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/tvm-dashboard-insights.md]:
```suggestion
> Machines with no alerts seen in the last 30 days do not count towards the exposure score of Threat & Vulnerability Management.
```

**Comment by e0i** [comment]
Pending follow-up.

**Comment by VLG17** [comment]
@e0i anything else needed from my side here?

**Pull request merged**